### ISSUE TITLE
p-selectButton does not have accessibility role defined

### ISSUE THREAD

**Issue by leonliang33** [primefaces/primeng#8119]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
p-selectButton inner components do not have accessibility role and name defined. This will cause screenreader to read "Group" rather than button

**Expected behavior**

**Minimal reproduction of the problem with instructions**
Have a p-selectButton and have screen reader on with ctrl + window + enter. Then navigate through each option and it will read "XXXXX group"

**What is the motivation / use case for changing the behavior?**
Accessibility requirements

**Please tell us about your environment:**
Edge / Chrome

* **Angular version:** 5.X
6

* **PrimeNG version:** 7.1.0

* **Browser:** [all ]
Chrome, Edge
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by yigitfindikli** [comment]
Fixed in https://github.com/primefaces/primeng/issues/8439
